python_wrapper/test.py

```diff
@@ -54,8 +57,8 @@ def run_test(self):
             try:
                 if result is 1:
                     self.handle_sat_case(f, cnf)
-                else:
-                    self.handle_unsat_case(f)
+                #else:
```

Comment:
I think you shold do
```
else:
    if not self.stealing:
        self.handle_unsat_case(f)
```
here, like this it just ignores it always! -> Bye bye correctness